### PR TITLE
Change: Pet item display scale

### PR DESCRIPTION
**Pull request by Thunderblade73**
allows scaling of the pet item display

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/config/features/misc/pets/PetConfig.java [hunk 1]
```diff
@@ -41,6 +42,12 @@ public class PetConfig {
         PetItemsDisplay.TIER_BOOST
     ));
 
+    @Expose
+    @ConfigOption(name = "Pet Item Scale", desc = "The scale at which the Pet Item will be displayed")
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/config/features/misc/pets/PetConfig.java]:
```suggestion
    @ConfigOption(name = "Pet Item Scale", desc = "The scale at which the Pet Item will be displayed.")
```

File: src/main/java/at/hannibal2/skyhanni/features/misc/PetItemDisplay.kt [hunk 2]
```diff
@@ -5,11 +5,13 @@ import at.hannibal2.skyhanni.events.GuiRenderItemEvent
 import at.hannibal2.skyhanni.utils.LorenzUtils
 import at.hannibal2.skyhanni.utils.RenderUtils.drawSlotText
 import at.hannibal2.skyhanni.utils.SkyBlockItemModifierUtils.getPetItem
+import net.minecraft.client.Minecraft
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 
 class PetItemDisplay {
 
     private val configList get() = SkyHanniMod.feature.misc.pets.petItemDisplay
+    private val scale get() = SkyHanniMod.feature.misc.pets.petItemDisplayScale
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/features/misc/PetItemDisplay.kt]:
```suggestion
    private val config get() = SkyHanniMod.feature.misc.pets
```
to fit with the majority of the codebase then use config.whatever later on

**Review by CalMWolfs — CHANGES_REQUESTED** [review]
2 small things

**Review by hannibal002 — CHANGES_REQUESTED** [review]
See Cal's suggestion

**Comment by Thunderblade73** [comment]
Now fixed.

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**